cli/lfc/src/test/java/org/lflang/cli/LfcCliTest.java

```diff
@@ -293,7 +293,7 @@ public void testGeneratorArgs(@TempDir Path tempDir) throws IOException {
       "rs",
       "--scheduler",
       "GEDF_NP",
-      "--threading",
+      "--single-threaded",
       "false",
```

Comment:
I'm confused. Are we supposed to write "false" here?

Reply:
Thanks for catching that - the "false" should have been deleted because single-threaded is now a switch. Will push a fix right now.